addons/l10n_ec/data/account_fiscal_position_template.xml

```diff
@@ -1,55 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
-    <record id="fp_companies" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Sociedades - personas juridicas</field>
-    </record>
-    <record id="fp_special_taxation_companies" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Contribuyentes especiales</field>
-    </record>
-    <record id="fp_public_companies" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Sector publico y ep</field>
-    </record>
-    <record id="fp_person_obligated_accounting" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Persona natural obligada a llevar contabilidad</field>
-    </record>
-    <record id="fp_person_leases" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Persona natural no obligada - arriendos</field>
-    </record>
-    <record id="fp_person_professional" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Persona natural no obligada - profesionales</field>
-    </record>
-    <record id="fp_person_rustic" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Persona natural no obligada - liquidaciones de compras</field>
-    </record>
-    <record id="fp_person_other" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Persona natural no obligadas - emite factura o nota de venta</field>
-    </record>
-    <record id="fp_foreing_company_local" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Empresa extranjera - venta local</field>
-    </record>
-    <record id="fp_foreing_person_local" model="account.fiscal.position.template">
-        <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Persona extranjera - venta local</field>
-    </record>
     <record id="fp_foreing_company_exports" model="account.fiscal.position.template">
         <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Empresa extranjera - exportacion</field>
+        <field name="name">Empresa extranjera</field>
```

Comment:
@pepetreshere Is it that difficult?  Because we could just map them to 0% for the simple cases.

Reply:
@jco-odoo I think I remember discussing this last year, and there were several combinations depending on the company responsibility type, customer responsibility type, etc, and we decided to leave it outside and let the user set that up if needed.

I don't know if we can at least do the "Foreign Transaction" one... probably only that one, and for the domestic operations, we will use standard logic (or user set up of fiscal position) for regular taxes and accounts + WTH configuration in the responsibility type, product config, and settings config. @pepetreshere

Reply:
If implemented we only need to implement the "Foreign Transaction" one. For local transaction the only change is in the wittholds which already have its own implementation in the new contributor type table and in the withhold wizard, no additional logic needed.

For Foreign Transactions the main problem is we have only one VAT 12% for local that should be mapped depending on the product type into "FOREIGN VAT 0% GOODS" or "FOREIGN VAT 0% SERVICES". For this an improvement in the map_tax method is needed to consider the nature of the product (the nature could be storable, consumable or services)

Another solution without coding is creating 2 local taxes (VAT 12% GOODS and VAT 12% SERVICES), but it might be confusing for many users as there will be 2 sale taxes with same code, only one of them suggested by default when creating new products. There are like 8 sale taxes to map (around 3 sales vat12% to map, 1 sales vat8% to map, 4 sales vat0% to map)